src/main/resources/application.properties

```diff
@@ -141,7 +141,7 @@ azure.retryTimeoutSeconds=3600
 #azure.monitoring.logCollectionConfigs[0].targetStorageAccountResourceId=resource id for storage account to store logs
 # ...
 
-rbs.enabled=false
+rbs.enabled=true
```

Comment:
What is this change for?

Reply:
This is more of a drive-by: When I was looking at the env variables to set for RBS, I saw this was disabled by default. But, this really should not be the case as we need RBS any GCP setup. But, this flag is only used in our status endpoint - determining whether or not to report an RBS outage as a critical system. 

I'll just flip it back since it's not really relevant and I don't think it hurts anything to leave it as false, as we're overriding this in all of our production systems (https://github.com/broadinstitute/terra-helmfile/blob/c8fa6bb2f1ceedb4df70c6fdff8aa6110971e0cc/values/app/datarepo/live/prod.yaml#L108).